### PR TITLE
Finalize prerelease disclosure

### PR DESCRIPTION
**Pull request by stigtsp**
- Finalize v1.0
- First signature

### CODE REVIEW

File: docs/pre-release-disclosure.md [hunk 1]
```diff
@@ -20,8 +19,8 @@ this information prior to the public announcement date of the vulnerability.
 
 ## Disclosure Agreement
 
-Members on the cpan-security@perl.org mailing list need to agree to the
-following:
+Members of need to agree to the following before getting access to information
```
Comment (timlegge) [inline docs/pre-release-disclosure.md:22]: maybe remove the "of" - Members need ...

File: docs/pre-release-disclosure.md [hunk 2]
```diff
@@ -20,8 +19,8 @@ this information prior to the public announcement date of the vulnerability.
...
+about undisclosed vulnerabilities trough private channels:
```
Comment (timlegge) [inline docs/pre-release-disclosure.md:23]: s/trough/through/

**Review by timlegge — CHANGES_REQUESTED** [review]
A couple of small changes

File: docs/pre-release-disclosure.md (hunk 1 above)
Comment (stigtsp) [reply docs/pre-release-disclosure.md]: Thx!

File: docs/pre-release-disclosure.md (hunk 2 above)
Comment (stigtsp) [reply docs/pre-release-disclosure.md]: Thx!

**Comment by timlegge** [comment]
Lgtm

**Pull request merged**